examples/nD_shapes.py

```diff
@@ -6,6 +6,8 @@
 from skimage import data
 import napari
 
+np.random.seed(0)
```

Comment:
Personally I would move this to right before the call to `np.random.uniform`, as it seems forlorn and out of place here.

Reply:
moved